localization/react-intl/src/app/components/UserInvalidInvitation.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "id": "userInvalidInvitation.heading",
+    "defaultMessage": "Failed to accept your invitation"
```

Comment:
"An error occurred while processing your invitation. Please try again and contact the support team if the issue persists."

Reply:
Also, is the backend returning some error that is not being shown here? If yes, then change to:
"An error occurred while processing your invitation. The server returned the error: {server_error}. Please try again and contact the support team if the issue persists."